sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOTest.java

```diff
@@ -146,4 +151,17 @@ public void test_FhirIO_failedPatientEverything() {
     PAssert.that(everythingResult.getPatientCompartments()).empty();
     pipeline.run();
   }
+
+  @Test
+  public void test_FhirIO_Export_invalidUri() {
+    final String invalidUri = "someInvalidUri";
+    pipeline
+        .apply(Create.of(Arrays.asList("resource1", "resource2")))
```

Comment:
What's the point of this line? Also just call FhirIO.exportResources(...) same as the other tests